.github/workflows/test.yml

```diff
@@ -1,34 +1,41 @@
-on: [push, pull_request]
 
-name: test
+on : [ push , pull_request ]
+
+name : test
 
 jobs:
-  test:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        deno: ['v1.x']
-        os: [ubuntu-latest, macOS-latest, windows-latest]
-    name: Deno ${{ matrix.deno }} test in ${{ matrix.os }}
-    steps:
-      - uses: actions/checkout@v2
-      - name: Setup deno
-        uses: denolib/setup-deno@v2
-        with:
-          deno-version: ${{ matrix.deno }}
+    test:
+        name : Deno ${{ matrix.deno }} test in ${{ matrix.os }}
+        runs-on : ${{ matrix.os }}
+        
+        strategy:
+            matrix:
+                deno : ['v1.x']
+                os : [ ubuntu-latest , macOS-latest , windows-latest ]
+        
+        steps:
+            
+            -   uses : actions/checkout@v2
+            
+            -
+                name : Setup deno
+                uses : denolib/setup-deno@v2
```

Comment:
Could you replace it with Deno official action?

Reply:
Didn't change it, but sure~